docker-compose/docker-compose-injistack/README.md

```diff
@@ -0,0 +1,154 @@
+# Inji Stack Setup
+
+This guide provides instructions for setting up and running Inji Stack.
+
+## Prerequisites
+- Docker and Docker Compose installed on your system
+- Git (to clone the repository)
+- Basic understanding of Docker and container operations
+### Building inji-web-proxy
+Before running the docker-compose, you need to build the inji-web-proxy image:
+
+```bash
+# Clone the repository
+git clone https://github.com/mosip/inji-web.git -b develop
```

Comment:
let's mention release-0.11.x branch for inji-web

Reply:
Done